src/components/StudyPane/InfoPane/Motif/RootBlock.tsx

```diff
@@ -40,22 +41,23 @@ export const RootBlock = ({
     if (rootBlockColor) {
       setColorFillLocal(rootBlockColor.colorFill);
       setTextColorLocal(rootBlockColor.textColor);
+      setBorderColorLocal(rootBlockColor.borderColor);
     }
   }, [ctxRootsColorMap]);
   
-  useEffect(() => {
-    const colorFill = descendants[0].colorFill;
-    const isSameColorFill = descendants.every((x) => x.colorFill === colorFill);
-    if (!isSameColorFill) {
-       setColorFillLocal(DEFAULT_COLOR_FILL);
-    } else if (colorFill != colorFillLocal) { setColorFillLocal(colorFill); }
-
-    const textColor = descendants[0].textColor;
-    const isSameTextColor = descendants.every((x) => x.textColor === textColor);
-    if (!isSameTextColor) {
-       setTextColorLocal(DEFAULT_TEXT_COLOR);
-    } else if (textColor != textColorLocal) { setTextColorLocal(textColor); }
-  }, [descendants]);
+  // useEffect(() => {
```

Comment:
is there a reason that we comment this out? We need this such that whenever there's a color change in one of the children, the color of the root block will be updated accordingly. So if any of the children/descendents have color change which doesn't match with others, then the root color will be set to default